website/docs/r/serviceendpoint_devops.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_serviceendpoint_devops"
+description: |-
+  Manages a Azure DevOps Service Connection service endpoint within Azure DevOps project.
+---
+
+# azuredevops_serviceendpoint_devops
+
+Manages a Azure DevOps Service Connection service endpoint within Azure DevOps. This service connection allows to trigger other subsequent builds with yaml block `task: RunPipelines@1`
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "project" {
+  project_name       = "Sample Project"
```

Comment:
`project_name` has been renamed to `name` in the latest master

Reply:
[x] renamed `project_name` to `name` in project definition in this example